### ISSUE TITLE
Diesel migrations not supported?

### ISSUE THREAD

**Issue by alexschrod** [adwhit/diesel-derive-enum#43]
Not sure if this is just me not doing things right, or if migrations aren't properly working, or what... but I put this in my `up.sql`:
```
CREATE TYPE unit_type AS ENUM ('weight', 'volume');

CREATE TABLE units (
  id SERIAL PRIMARY KEY,
  base_id INT REFERENCES units(id) ON DELETE CASCADE,
  base_multiplier DECIMAL,
  name VARCHAR NOT NULL,
  type unit_type NOT NULL
);
```
and after running the migration, Diesel puts this into the generated `schema.rs`:
```
table! {
    units (id) {
        id -> Int4,
        base_id -> Nullable<Int4>,
        base_multiplier -> Nullable<Numeric>,
        name -> Varchar,
        #[sql_name = "type"]
        type_ -> Unit_type,
    }
}
```
Already there's a problem. Based on your README, which says
> By default, the Postgres and Diesel internal types are inferred from the name of the Rust enum. Specifically, we assume MyEnum corresponds to my_enum in Postgres and MyEnumMapping in Diesel. (The Diesel type is created by the plugin, the Postgres type must be created in SQL).

I would have expected the `unit_type` to become `UnitType`, not `Unit_type`. Then I thought, maybe I have to define `UnitType` myself. So I created this before running:
```
#[derive(Debug, diesel_derive_enum::DbEnum)]
#[PgType = "unit_type"]
#[DieselType = "Unit_type"]
pub enum UnitType {
    Weight,
    Volume,
}
```
Running the migration now, it's still the same as above. If I manually change the `type_ -> Unit_type` line to `type_ -> crate::database::models::Unit_type`, everything works, but this is a relatively fragile solution, as the `schema.rs` file gets overwritten by the migration tool, so I'd have to make that change every time.

It's not the end of the world, but it would be nice if it *just worked™*.

**Comment by adwhit** [comment]
Hi, this is really more to do with the way `diesel` works than this crate. You might find the following article helpful for getting it working: https://atsuzaki.com/blog/diesel-enums/

TLDR you need to modify `diesel.toml` to include:
```
import_types = ["diesel::sql_types::*", "crate::database::models::*"]
```

**Comment by alexschrod** [comment]
Thanks a lot for that information, it would've taken me a lot longer to reach a good solution without it!
